src/ophys_etl/transforms/suite2p_container.def

```diff
@@ -0,0 +1,44 @@
+Bootstrap: docker
+From: continuumio/miniconda3:4.8.2
+
+%post
+    # these can be set during build like
+    # sudo SINGULARITYENV_SUITE2P_TAG=0.7.5 singularity build ...
+    SUITE2P_TAG=${SUITE2P_TAG:-0.7.5}
+    OPHYS_ETL_TAG=${OPHYS_ETL_TAG:-master}
+
+    export PATH=/opt/conda/bin:${PATH}
+
+    # initialize conda
+    CONDA_BASE=$(conda info --base)
+    . $CONDA_BASE/etc/profile.d/conda.sh
+
+    # make this variable available at runtime
+    echo "export CONDA_BASE=\"${CONDA_BASE}\"" >> $SINGULARITY_ENVIRONMENT
+
+    # install Suite2P
+    apt-get install -q -y libgl1-mesa-glx
```

Comment:
I would also `&& rm -rf /var/lib/apt/lists/*` so that the downloaded version of the dependency doesn't get cached.